src/layouts/EditContactUs.jsx

```diff
@@ -774,27 +783,49 @@ const EditContactUs = ({ match }) => {
 
   return (
     <>
-      {showDeletedText && !isEmpty(deletedFrontMatter) && (
+      {!isEmpty(deletedFrontMatter) && (
         <GenericWarningModal
+          isOpen={isRemovedOpen}
+          onClose={onRemovedClose}
           displayTitle="Removed content"
           displayText={`Some of your content has been removed as it is incompatible with the new Isomer format. No changes are permanent unless you press Save on the next page.<br/>${displayDeletedFrontMatter(
             deletedFrontMatter
           )}`}
-          onProceed={() => {
-            setShowDeletedText(false)
-          }}
-          proceedText="Acknowledge"
-        />
+        >
+          <Button onClick={onRemovedClose}>Acknowledge</Button>
+        </GenericWarningModal>
       )}
       {itemPendingForDelete.id && (
-        <DeleteWarningModal
-          onCancel={() => setItemPendingForDelete({ id: null, type: "" })}
-          onDelete={() => {
-            deleteHandler(itemPendingForDelete.id)
+        <GenericWarningModal
+          isOpen={isDeleteOpen}
+          onClose={() => {
             setItemPendingForDelete({ id: null, type: "" })
+            onDeleteClose()
           }}
-          type={itemPendingForDelete.type}
-        />
+          displayTitle={`Delete ${itemPendingForDelete.type} section`}
+          displayText={`Are you sure you want to delete ${itemPendingForDelete.type}?`}
+        >
+          <Button
+            variant="ghost"
```

Comment:
this variant is from chakra and might not be what was intended!

Reply:
Changed in https://github.com/isomerpages/isomercms-frontend/pull/941/commits/a4e792aaffbb26913a0cc9a40493c17faf42a5d7!